references/kvm-admin-api/apiproxy/policies/SpikeArrest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<SpikeArrest continueOnError="false" enabled="true" name="SpikeArrest">
```

Comment:
Trivial for consistency:

rename: SA-SpikeArrest

remove: 
* `continueOnError="false" enabled="true"`
*  DisplayName
*  Properties